### DISCUSSION TITLE
How to update from V1.19 to V1.20 with keeping the existing settings and database (historical data)

### DISCUSSION THREAD

**Discussion by killerbeat12** [frangoteam/FUXA#1417] [Q&A]
Hi, i am missing the update manual to go from an existing V1.19 to V1.20 and keep all project files and historical data.
I am using the simple installation method which means only extracting the zip file and following with npm install and npm start
(Windows Server 2022)

Please help? Thanks

**Comment by unocelli** [comment]
Hi, update should not lose existing values, however if you make a backup in FUXA/server of the following folders:
-  _appdata
- _db
- _images
- _logs
- _pkg
- reports
